frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -821,6 +821,10 @@ export function ContainersSection(props: { resource: KubeObjectInterface | null
     return containers;
   }
 
+  function getEphemeralContainers() {
+    return resource?.spec.ephemeralContainers || [];
```

Comment:
Notice that the getInitContainers also has a ? after the spec. Let's add that as a defensive measure.

Reply:
done!